insights/specs/default.py

```diff
@@ -180,6 +180,7 @@ def is_ceph_monitor(broker):
     dirsrv = simple_file("/etc/sysconfig/dirsrv")
     dirsrv_access = glob_file("var/log/dirsrv/*/access")
     dirsrv_errors = glob_file("var/log/dirsrv/*/errors")
+    disk_type = glob_file("/sys/block/*/queue/rotational")
```

Comment:
@smalleni you also need to add a RegistryPoint for the `disk_type` spec to the file `insights/specs/__init__.py`.